src/main/java/com/example/NiRi/PasswordJWT.java

```diff
@@ -1,23 +1,22 @@
 package com.example.NiRi;
 
-
+import com.example.NiRi.modules.User;
 import io.jsonwebtoken.Jwts;
 import io.jsonwebtoken.SignatureAlgorithm;
 import org.springframework.stereotype.Component;
-
 import java.util.Date;
 
 @Component
 public class PasswordJWT {
 
-    private static final String SECRET_KEY = "yourSecretKey"; // Replace with a secure key
+    private static final String SECRET_KEY = "SECRET_KEY";
```

Comment:
use the secret key from your application.properties file with @Value

Reply:
Sure! I´l make the necessary changes.